packages/kafka-iam-auth/src/index.ts

```diff
@@ -49,7 +49,7 @@ const handleExit = (consumer: Consumer): void => {
 const initConsumer = async (
   config: ConsumerConfig,
   topics: string[],
-  consumerHandler: (message: KafkaMessage) => Promise<void>
```

Comment:
@MalpenZibo FYI: we changed the type signature of the consume handler in order to access the partition number to log it